example_template.md

```diff
@@ -0,0 +1,104 @@
+**Table of Contents**
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+- [Run the EXAMPLE NAME](#run-the-example-name)
+  - [🗝️ Pre requirements](#Pre-requirements)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌀 Flow Diagram](#flow-diagram)
+  - [🌟 Results](#results)
+  - [🧞 OPTIONAL: EXTRA KNOWLEDGE FROM THIS EXAMPLE](#optional)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Run the EXAMPLE NAME
+
+Write a short description of
+1. What is this?
+2. What is the expected outcome?
+3. What the user will learn?
+
+If you want to run the example with Docker, check the instruction at the end of this README.  
+
+## 🗝️ Pre requirements
+
+Outline in bullet points anything the user is expected to have before diving in. For example:
+
+1. You have a working Python 3.8 environment. 
+2. We recommend creating a [new python virtual envoriment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency clashing.   
+3. You have at least 8GB of free space on your hard drive. 
+
+### Install requirements
+
+In order to install all the Python libraries required you can run the following in your terminal:
+
+```
+pip install -r requirements.txt
+```
+
+## 🔮 Overview of the files
+
+Add a list with all files in the example:
+
+* `index.yml`: YAML file for indexing
+* `query.yml`: YAML file for querying
+* `encoder.yml`: YAML file for encoder pod
+* `/workspace: Directory that stores indexed files (embeddings and documents). Automatically created after the first indexing.
+
+## 🏃 Run the Flows
```

Comment:
These instructions explain how to build the example yourself and deploy it with Python. If you want to skip the building skips and just run the app, check out the Docker section below.

Reply:
I added it also. There was another comment from Agnes I didn't see before so it's all now in the latest PR